app.yaml

```diff
@@ -1,6 +1,18 @@
-runtime: go112
+runtime: python37
```

Comment:
I wasn't able to get the `go` runtime to only host static files

Reply:
That's weird! How come? I didn't actually realise we weren't using Hugo at all for the _serving_, only the site building, so I guess it doesn't matter! What do people generally use for serving static sites from AppEngine then?

Reply:
From what I can tell, most use python, although in our case the run-time shouldn't matter because there's no web server defined in our app anywhere. I bet the python runtimes have a built-in gunicorn or something similar that go doesn't have.

Reply:
Good call! Well, if this works, I say make it so!